### ISSUE TITLE
[ML-Kit] Result is not compatible with type ScanResult

### ISSUE THREAD

**Issue by iBobik** [capacitor-community/barcode-scanner#251]
**Describe the bug**
BarcodeScanner.start's callback should return object compatible with (ScanResult interface](https://github.com/capacitor-community/barcode-scanner/blob/ml-kit/src/definitions.ts#L160)
But actually it returns object like this:
```
{ hasContent: true, content: 'https://app.sarisskahradnacesta.sk/p/343583', bounds: '9 62 671 838' }
```

**To Reproduce**

1. Use BarcodeScanner.start on an Android
2. console.log in a callback
3. See what it returned

**Expected behavior**
Result will be compatible with a typing.

**Version**
5.0.0-beta.1

**Smartphone (please complete the following information):**

- Device: Galaxy A13
- OS: Android 13
- Browser Capacitor
- Version 5.0.4

**Comment by thegnuu** [comment]
You are right, I will fix this in the next release. There are still some debug outputs ;)
